.github/workflows/sdk-update-api-spec.yaml

```diff
@@ -0,0 +1,41 @@
+name: SDK Update API from from Spec
+
+on:
+  schedule:
+    - cron: '0 4 * * *'
+  workflow_dispatch:
+
+jobs:
+  api-spec-update:
+    runs-on: ubuntu-20.04
+    if: ${{ github.repository == 'jellyfin/jellyfin-sdk-kotlin' }}
+    steps:
+      - name: Setup Java
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'adopt'
+          java-version: 11
+      - name: Setup Gradle cache
+        uses: actions/cache@v2.1.4
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+      - name: Set Stable API Version EnvVar
+        run: |
+          VERSION=$(curl -sL https://repo.jellyfin.org/releases/openapi/jellyfin-openapi-stable.json | jq -r .info.version)
+          echo "STABLE_API_VERSION=${VERSION}" >> $GITHUB_ENV
```

Comment:
I think this approach is better then the previous one although it technically could still return another version as the Gradle task. The changes of that are low though so I think it's fine.

Reply:
The version would have to change in between the two steps for that, which I think can be ignores (yet has to be noted for sure)